Dockerfile_sheduler

```diff
@@ -0,0 +1,17 @@
+FROM python:3.8
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+ENV SERVICE=${run_service:-backend}
+ENV QUEUES=${queues:-celery}
+
+WORKDIR /opt/djangoTrading/
+
+RUN pip install --upgrade pip
+COPY run_scheduler.sh /opt/djangoTrading/
+COPY requirements.txt /opt/djangoTrading/
+RUN pip install -r requirements.txt --no-cache-dir
+
+COPY . /opt/djangoTrading/
+
+ENTRYPOINT [ "/bin/bash", "/opt/djangoTrading/run_scheduler.sh" ]
```

Comment:
No, you have no need to separate Dockerfiles. Main aim is to save one Dockerfile and make it reusable for celery and django using [entrypoint command](https://docs.docker.com/compose/compose-file/compose-file-v3/#entrypoint) in docker-compose